src/parsley/main.js

```diff
@@ -113,11 +113,10 @@ $.each('addError updateError'.split(' '), function (i, method) {
 // ### PARSLEY auto-binding
 // Prevent it by setting `ParsleyConfig.autoBind` to `false`
 if (false !== window.ParsleyConfig.autoBind) {
-  $(function () {
```

Comment:
This changes the timing of the autobind. It should be remain the same, i.e. after loading is complete instead of instantly.

Reply:
Done. Please review